builder.go

```diff
@@ -103,6 +107,12 @@ func (b *Builder) Build() (*platform.BuildMetadata, error) {
 		}
 	}
 
+	b.Logger.Debug("Copying BOM files")
+	err = b.copyBOMFiles(config.LayersDir, bomFiles)
```

Comment:
Do we want to put the platform API gate here, to make it more readable?

Reply:
_Fixed in latest commit._